src/java.base/windows/native/libjava/java_props_md.c

```diff
@@ -634,6 +634,7 @@ GetJavaProperties(JNIEnv* env)
         LCID userDefaultUILCID = MAKELCID(userDefaultUILang, SORTIDFROMLCID(userDefaultLCID));
 
         {
+            HANDLE hStdIn;
```

Comment:
Instead of creating a new variable, `hStdOutErr` could be reused with a proper name.

Reply:
OK I merged the local variables into a single one named `hStdHandle`. I'm not sure that's the best name. I surmise that the `h` prefix is "Hungarian Notation" to indicate that it's a handle, though it makes the name kind of redundant.